src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,5 @@
+$transition-duration:300ms;
+
 .catalog {
```

Comment:
Same case with separate files, there must be a folder for blocks. 
By the way, you forgot about 'header' block, check p4 of the main task (Smoothly (during 300ms) change navigation link text color to #00acdc)